### PR TITLE
Added two file

### PR DESCRIPTION
**Pull request by mumbereh**
Added index.html & style.css file

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @mumbereh , 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not reviewed at all, please review the project and submit it again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @mumbereh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that s not been reviewed at all, please review your project and please read all your project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Free codecamp servy form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body background="https: //cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg">
```
Comment (mumbereh) [inline index.html:10]: Shift the background image from index.html file to style.css file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        <input type="radio" value="Not sure" name="microverse">Not sure<br>
+        What is your favorite feature of freeCodeCamp?<br>
+        <select id="selection">
+            <option>Select an option</option>
+            <option>Challenges</option>
+            <option>Projects</option>
+            <option>Community</option>
+            <option>open source</option><br>
+        </select><br><br>
+            What would you like to see improved? (Check all that apply)<br>
```
Comment (mumbereh) [inline index.html:55]: Apply section on check element.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            <input type="checkbox" value="Challenges">Challenges<br>
+            <input type="checkbox" value="Open Source Community">Open Source Community<br>
+            <input type="checkbox" value="Gitter help rooms">Gitter help rooms<br>
+            <input type="checkbox" value="Videos">Videos<br>
+            <input type="checkbox" value="City Meetups">City Meetups<br>
+            <input type="checkbox" value="Wiki">Wiki<br>
+            <input type="checkbox" value="Forum">Forum<br>
+            <input type="checkbox" value="Additional Courses">Additional Courses<br>
+            Any comments or suggestions?<br>
+            <textarea width="400" height="200">
```
Comment (mumbereh) [inline index.html:68]: Add placeholder to the textarea in a professional way.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+         Enter your comment here----
+            </textarea><br><br>
+            
+        <input type="submit" id="submit" value ="submit">
```
Comment (mumbereh) [inline index.html:72]: Check input to button

**Review by mumbereh — COMMENTED** [review]
hi @mumbereh

Good job so far!
You still need to work on some issues , but you are almost there!

Required Changes

Add linters
Make your commit history professional
Make PR descriptive
Make your README file professional
Happy coding

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @mumbereh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Good use of summary template ✔️
Good use of inline comments ✔️


## Required Changes ♻️
- [ ] Kindy, state the reason(s) for the suggested changes in a friendly manner, it may be obvious to you and not so for others. 👍

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mumbereh — COMMENTED** [review]
Hi @mumbereh
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there!
Kindly do all styling from style.css file


143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91

- [ ] 1. Make the title of your repo descriptive and professional. This gives more understanding to the user/reader
- [ ] 2. kindly shift styling from index.html to style.css. check line 10 of index.html. transport that background styling to style.css file.
- [ ] 3. Add README file to your project. this makes the project author more professional since it holds the summery of the project

**Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by Strangeal — APPROVED** [review]
Hi @mumbereh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**